packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/README.md

```diff
@@ -9,6 +9,17 @@ The Bar Chart Viewer...
 - allows users to view bar chart data (x-axis: categorical values, y-axis:
   numerical values for each category)
 
+## Props
+
+- `barStyles` _(object)_ An object containing properties for the individual bar styles. `padding` is the available configurable style and sets the space between each bar. It defaults to `0.25`.
+- `data` _(array)_ An array of objects. Each object must contain a string label and a number value. A string color value is optional for each bar and can be set to either a string variable name from the [Zooniverse theme](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-grommet-theme) or any hex value. A color variable name is preferred from the theme since it has been approved by the Zooniverse designer and tested for accessibility. This should be set by the data property in the location JSON of the subject. See the section on [JSON file](#JSON_file) for more information. Required.
+- `margin` _(object)_ An object of the numerical values for `top`, `bottom`, `left`, `right`. This sets the SVG space outside of the axes lines. This is configurable via the subject location JSON's options property (See the section on [JSON file](#JSON_file)). The amount of space necessary can vary based on the data, what the axes tick labels are, what the axes labels are, etc. A default of `{ bottom: 40, left: 40, right: 10, top: 10 }` is set. Bottom and left margin is greater since that is where the x-axis and y-axis are positioned for the chart.
```

Comment:
I think the default value should be specified at the beginning of the definition, not buried in the text.